core/registry.js

```diff
@@ -16,12 +16,13 @@ goog.provide('Blockly.registry');
 goog.requireType('Blockly.blockRendering.Renderer');
 goog.requireType('Blockly.Field');
 goog.requireType('Blockly.IToolbox');
+goog.requireType('Blockly.utils.toolbox');
 
 
 /**
  * A map of maps. With the keys being the type and name of the class we are
  * registering and the value being the constructor function.
- * Ex: {'field': {'field_angle': Blockly.FieldAngle}}
+ * ex: {'field': {'field_angle': Blockly.FieldAngle}}
```

Comment:
I think what you're after here is `e.g.`
https://english.stackexchange.com/questions/16197/whats-the-difference-between-e-g-and-ex